Plugins/SequencePlugin/Source/SequencePlugin/Public/Util/SequenceSupport.h

```diff
@@ -335,10 +335,6 @@ class SEQUENCEPLUGIN_API USequenceSupport : public UObject
 
 	static FString JsonToSimpleString(const TSharedPtr<FJsonObject>& JsonData);
 
-	//indexer response extraction functions used to parse response into a frontend usable form!
-	static FUpdatableHistoryArgs ExtractFromTransactionHistory(FString MyAddress, FSeqGetTransactionHistoryReturn TransactionHistory);
-	static FUpdatableItemDataArgs ExtractFromTokenBalances(FSeqGetTokenBalancesReturn TokenBalances);
```

Comment:
What were these used for? They sound like they might be useful for users, but I don't have as much context

Reply:
Looks like those functions were only referenced from another class called `USystemDataBuilder`, but this class was never used anywhere else, so probably a deprecated system.